docs/UserGuide.md

```diff
@@ -66,13 +70,34 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 - If you are using a PDF version of this document, be careful when copying and pasting commands that span multiple lines as space characters surrounding line-breaks may be omitted when copied over to the application.
   </box>
 
-### Adding new students : `add`
+### Adding new students : `add_student`
+
+Adds a new student contact with all the details that have been specified by the user to the relevant module and tutorial class
+
+Format: `/add_student /name NAME /email EMAIL /id STUDENT_ID /tc TUTORIAL_CLASS`
+
+- The following parameters to add a student contact are supported:
+    1. Name
+    2. Email
+    3. Student ID
+    4. Tutorial class
```

Comment:
In v1.2, we have previously discussed that we will not be supported adding students to a tutorial class (at least for the MVP). Nevertheless, this is our ultimate end goal. However, you might want to add in a point about throwing an error if an invalid tutorial class is specified.